python/dgl/transform.py

```diff
@@ -147,20 +148,96 @@ def segmented_knn_graph(x, k, segs):
     g = convert.graph(adj)
     return g
 
-def to_bidirected(g, readonly=None, copy_ndata=True,
-                  copy_edata=False, ignore_bipartite=False):
-    r"""Convert the graph to a bidirected one.
+def to_bidirected(g, readonly=True):
+    r""" Convert the graph to a bidirected one.
 
-    For a graph with edges :math:`(i_1, j_1), \cdots, (i_n, j_n)`, this
-    function creates a new graph with edges
-    :math:`(i_1, j_1), \cdots, (i_n, j_n), (j_1, i_1), \cdots, (j_n, i_n)`.
+    The function generates a new graph with no node/edge feature.
+    If g has an edge for i->j but no edge for j->i, then the
+    returned graph will have both i->j and j->i.
 
     For a heterograph with multiple edge types, we can treat edges corresponding
     to each type as a separate graph and convert the graph to a bidirected one
     for each of them.
 
     Since **to_bidirected is not well defined for unidirectional bipartite graphs**,
     an error will be raised if an edge type of the input heterograph is for a
+    unidirectional bipartite graph.
+
```

Comment:
"Node and edge features are ignored."

Also, I guess it makes sense to copy node features (unless @jermainewang is still working on the frame indexing idea, in which case we don't need it).